### PR TITLE
media-sound/guitarix: new package

### PR DESCRIPTION
**Pull request by puleglot**
guitarix is a virtual guitar amplifier for Linux

Ebuild by me with some additions from ebuild by Reva Denis and ebuild from gentoostudio overlay.

Closes: https://bugs.gentoo.org/254707

### CODE REVIEW

File: media-libs/zita-resampler/zita-resampler-1.6.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	fi
+}
+
+src_install() {
+	emake -C source DESTDIR="${D}" PREFIX="${EPREFIX}/usr" LIBDIR="${EPREFIX}"/usr/$(get_libdir) install
+	if use tools; then
+		emake -C apps DESTDIR="${D}" PREFIX="${EPREFIX}/usr" install
+	fi
+
+	default
```
Comment (gyakovlev) [inline media-libs/zita-resampler/zita-resampler-1.6.2.ebuild]: isn't running default here kind of redundant?
it runs
`emake DESTDIR="${D}" install`

I haven't tested, just asking )

Comment (puleglot) [reply media-libs/zita-resampler/zita-resampler-1.6.2.ebuild]: There is no Makefile in ${S}, but you are right, it is better to replace default with einstalldocs.

**Comment by gyakovlev** [comment]
```
QA: other
QA Notice: Files built without respecting LDFLAGS have been detected
 Please include the following list of files in your report:
/usr/lib64/libgxw.so
/usr/lib64/libgxw.so.0.1
/usr/lib64/libgxwmm.so
/usr/lib64/libgxwmm.so.0.1
```

can you look at this? otherwise looks fine and works great.

**Comment by puleglot** [comment]
Thanks! Should be fixed.

**Comment by gyakovlev** [comment]
pushed, thanks!

**Pull request closed without merging**